src/main/java/com/server/ResponseBodyBuilder.java

```diff
@@ -13,9 +13,9 @@
         bodyHashtable.put("/coffee", "I'm a teapot");
     }
 
-    public String getBody(String path, String method) {
-        if(this.requestRouter.getResponseCode(path, method) != 0){
-            String value = (String)this.bodyHashtable.get(path);
+    public String getBody(RequestParams requestParams) {
+        if(this.requestRouter.getResponseCode(requestParams.getPath(), requestParams.getMethod()) != 0){
```

Comment:
What are your thoughts on having the `RequestRouter.getResponseCode()` method accept a `RequestParam` object, instead of `path` and `method`?

Reply:
Makes sense, updated method signature for consistency with other methods that have changed to take in a RequestParams object. I had initially left it there as is to pass in only what it needs but we want to avoid a Data Clump. Thank you!